test/TestProjects/Customizations-Typespec/Generated/Docs/CustomizationsInCadlClient.xml

```diff
@@ -1,309 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <doc>
   <members>
-    <member name="RoundTripAsync(RequestContent,RequestContext)">
```

Comment:
why is this doc now empty?

Reply:
Because this client now does not have a public ctor, we do not know a way to construct it and all the content in this doc is depending on the public ctor, therefore the doc is empty by design.

Reply:
Does this mean it has a method from its parent to retrieve it?  If so are we not able to get that information to still provide an example?

Reply:
The generator will not know that. The generator is taking the accessibility on customized code for the constructors, it only sees the client no longer has public constructors now. There might be a way to get the client in other ways, like those are written in customization code, but the generator will not know that.
Therefore in order not to generate invalid example code (since the ctor is not public), I choose we just generate nothing for the documents.

Reply:
Let see if we can find this information.  It seems doable to determine if there is a method or property on any other client that returns this client and if so we know how its constructed.